src/canvas_test.c

```diff
@@ -0,0 +1,158 @@
+#include "minunit.h"
+#include "canvas.h"
+
+static int cols = 2;
+static int rows = 3;
+static char load_str[] = "012345";
+
+static Canvas *c1, *c2;
+
```

Comment:
```suggestion

// Begin canvas_bootstrapping tests
```

Below, the jump to canvas_main tests confused me, so I suggested a comment. Adding this one in to be consistent.